### PR TITLE
[FIX] Some apps were not correctly enabled during startup in HA environments

### PR DESCRIPTION
**Pull request by thassiov**
## Proposed changes

Updates Apps-Engine's version with the latest bug fixes and adjusts properties in the apps' db collection that are wrongly set by a bug when running Rocket.Chat in HA mode

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves a current function)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Hotfix (a major bugfix that has to be merged asap)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat) doc
- [x] I have signed the [CLA](https://cla-assistant.io/RocketChat/Rocket.Chat)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] I have added necessary documentation (if applicable)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by d-gubert** [comment]
THE MOST WANTED PR OF THE YEAR

**Comment by sampaiodiego** [comment]
NO WAY

**Comment by d-gubert** [comment]
It looks good to me, but I'll wait for @sampaiodiego's more certified look on the Migration 😬

File: server/startup/migrations/v212.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { MongoInternals } from 'meteor/mongo';
+
+import { Migrations } from '../../../app/migrations';
+
+Migrations.add({
+	version: 212,
+	async up() {
```
Comment (sampaiodiego) [inline server/startup/migrations/v212.js:7]: migration cannot be `async` as it returns immediately and the runner is not awaiting the result.

File: server/startup/migrations/v212.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+		const { mongo } = MongoInternals.defaultRemoteCollectionDriver();
+		const apps = mongo.db.collection('rocketchat_apps');
+
+		await apps.update({
```
Comment (sampaiodiego) [inline server/startup/migrations/v212.js:11]: removing `async` from function, you can use `Promise.await` here, so it will await using Fibers :wink:

**Review by sampaiodiego — CHANGES_REQUESTED** [review]

File: server/startup/migrations/v212.js (hunk 2 above)
Comment (thassiov) [reply server/startup/migrations/v212.js]: done. thanks for the tip

**Review by sampaiodiego — APPROVED** [review]
I don't know about the logic being the migration, like why "initialiazed" should become "manually_disabled".. but the migration itself looks good

**Comment by thassiov** [comment]
@sampaiodiego it's about _correcting_ the state in the app's record, because the bug leaves the affected app as `initialized` and this state is not valid (that's why the apps appear as "disabled"). The "real/official" states should be "enabled" or "disabled", so with this migration we are cleaning after the bug, essentially

File: server/startup/migrations/v212.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+import { Migrations } from '../../../app/migrations';
+
+Migrations.add({
+	version: 212,
+	up() {
+		const { mongo } = MongoInternals.defaultRemoteCollectionDriver();
+		const apps = mongo.db.collection('rocketchat_apps');
+
+		Promise.await(apps.update({
```
Comment (sampaiodiego) [inline server/startup/migrations/v212.js:11]: is it too late (or picky from me) to ask to change to `updateMany`? `update` is currently deprecated 😬

Comment (sampaiodiego) [reply server/startup/migrations/v212.js]: I changed to `updateMany` =)

**Review by sampaiodiego — APPROVED** [review]

**Pull request merged**